domain/src/test/java/org/oppia/android/domain/oppialogger/analytics/CpuPerformanceSnapshotterTest.kt

```diff
@@ -231,6 +240,127 @@ class CpuPerformanceSnapshotterTest {
     assertThat(fakePerformanceMetricsEventLogger.noPerformanceMetricsEventsPresent()).isTrue()
   }
 
+  @Test
+  fun testSnapshotter_onCreate_moveToForegroundBeforeCutOff_logsCpuUsageInForegroundAfterDelay() {
+    applicationLifecycleObserver.onCreate()
+    testCoroutineDispatchers.runCurrent()
+    // clearing up all app startup performance metrics: apk size and storage usage.
```

Comment:
Nit: line comments should be capitalized for code parity (ditto elsewhere in this file).

Reply:
done.